src/components/Navmenu.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { uniq, includes } from 'lodash';
+import CloseSVG from 'svg/Close';
+
+class Navmenu extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      nav: []
+    };
+  }
+
+  menuClassName = () => {
+    const baseClassName = `coa-Navmenu`;
+    return this.props.isOpen ? `${baseClassName}--open` : baseClassName;
+  }
+
+  focusOnClose = () => {
+    this.refs.closeTrigger.focus()
+  }
+
+  componentDidMount () {
+    const endpoint = [
```

Comment:
thinking we should start defining our endpoint urls in one place, since these will likely be changing and we’ll likely only want to update these urls in one place.

Reply:
cool, i'll create a file at `src/constants/endpoints` and import the string from there.